src/OpenEventSourcing/Extensions/OpenEventSourcingBuilderExtensions.cs

```diff
@@ -31,17 +31,40 @@ public static IOpenEventSourcingBuilder AddEvents(this IOpenEventSourcingBuilder
         {
             if (builder == null)
                 throw new ArgumentNullException(nameof(builder));
-
+            
```

Comment:
Can we remove the commented out code? or are we planning to use this in the future?